pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -202,6 +207,8 @@ const (
 	// PipelineRunSpecStatusCancelled indicates that the user wants to cancel the task,
 	// if not already cancelled or terminated
 	PipelineRunSpecStatusCancelled = "PipelineRunCancelled"
+
+	PipelineRunSpecStatusPending = "PipelineRunPending"
```

Comment:
nit: I think it's worth adding a comment here similar to PipelineRunSpecStatusCancelled describing its purpose.  Something like:

```go
// PipelineRunSpecStatusPending indicates that the user wants to postpone starting a PipelineRun
// until some condition is met
```